tasks/setup_plugins.yml

```diff
@@ -21,6 +21,8 @@
     update: True
   tags: [ 'role::checkmk_agent:plugins:get' ]
   delegate_to: '{{ checkmk_agent__git_dest_host }}'
+  become: '{{ False if (checkmk_agent__git_dest_host in ["localhost"]) else True }}'
```

Comment:
Just a habit because there could be more cases. `checkmk_agent__git_dest_host == "localhost"` would also so. Should I change it?